pkg/apis/pipeline/v1alpha1/stepaction_types.go

```diff
@@ -121,6 +121,12 @@ type StepActionSpec struct {
 	// +optional
 	// +listType=atomic
 	Results []StepActionResult `json:"results,omitempty"`
+	// SecurityContext defines the security options the Step should be run with.
```

Comment:
```suggestion
	// SecurityContext defines the security options the StepAction should be run with.
```

Reply:
But StepAction is actually merged to a Step? StepAction will not `run`. I hesitated a bit when copying the docstring 😄 . Let me know if we should still use StepAction instead

Reply:
Yes. I think that's ok. Lets leave it as is since we eventually run the Step not the StepAction.